<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="6" ignored="0" total="11" passed="5">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="78" started-at="2022-02-28T04:54:25 YAKT" finished-at="2022-02-28T04:54:25 YAKT">
    <groups>
    </groups>
    <test name="Default test" duration-ms="78" started-at="2022-02-28T04:54:25 YAKT" finished-at="2022-02-28T04:54:25 YAKT">
      <class name="online_tusur.unit_online_tusur.TestFullStudent$GetFullNameTest">
        <test-method status="PASS" signature="test_fullName(java.lang.String,online_tusur.unit_online_tusur.Student)[pri:0, instance:online_tusur.unit_online_tusur.TestFullStudent$GetFullNameTest@4af45442, instance params:[]]" name="test_fullName" duration-ms="1" started-at="2022-02-28T04:54:25 YAKT" data-provider="fullNameProvider" finished-at="2022-02-28T04:54:25 YAKT">
          <params>
            <param index="0">
              <value>
                <![CDATA[Ivanov Ivan]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[online_tusur.unit_online_tusur.Student@429c5afb]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test_fullName -->
        <test-method status="FAIL" signature="test_fullName(java.lang.String,online_tusur.unit_online_tusur.Student)[pri:0, instance:online_tusur.unit_online_tusur.TestFullStudent$GetFullNameTest@4af45442, instance params:[]]" name="test_fullName" duration-ms="2" started-at="2022-02-28T04:54:25 YAKT" data-provider="fullNameProvider" finished-at="2022-02-28T04:54:25 YAKT">
          <params>
            <param index="0">
              <value>
                <![CDATA[Petrova-Sergeeva Alla-Victoria]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[online_tusur.unit_online_tusur.Student@2e820c6a]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Petrova-sergeeva Alla-victoria] but found [Petrova-Sergeeva Alla-Victoria]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Petrova-sergeeva Alla-victoria] but found [Petrova-Sergeeva Alla-Victoria]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at online_tusur.unit_online_tusur.TestFullStudent$GetFullNameTest.test_fullName(TestFullStudent.java:83)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:77)
at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:15)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test_fullName -->
      </class> <!-- online_tusur.unit_online_tusur.TestFullStudent$GetFullNameTest -->
      <class name="online_tusur.unit_online_tusur.TestFullStudent$GetFirstNameTest">
        <test-method status="FAIL" signature="test_getFirstName(java.lang.String,online_tusur.unit_online_tusur.Student)[pri:0, instance:online_tusur.unit_online_tusur.TestFullStudent$GetFirstNameTest@2e645fbd, instance params:[]]" name="test_getFirstName" duration-ms="2" started-at="2022-02-28T04:54:25 YAKT" data-provider="firstNameProvider" finished-at="2022-02-28T04:54:25 YAKT">
          <params>
            <param index="0">
              <value>
                <![CDATA[Alla-Victoria]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[online_tusur.unit_online_tusur.Student@565bbe8f]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Alla-victoria] but found [Alla-Victoria]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Alla-victoria] but found [Alla-Victoria]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at online_tusur.unit_online_tusur.TestFullStudent$GetFirstNameTest.test_getFirstName(TestFullStudent.java:49)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:77)
at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:15)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test_getFirstName -->
        <test-method status="FAIL" signature="test_getFirstName(java.lang.String,online_tusur.unit_online_tusur.Student)[pri:0, instance:online_tusur.unit_online_tusur.TestFullStudent$GetFirstNameTest@2e645fbd, instance params:[]]" name="test_getFirstName" duration-ms="1" started-at="2022-02-28T04:54:25 YAKT" data-provider="firstNameProvider" finished-at="2022-02-28T04:54:25 YAKT">
          <params>
            <param index="0">
              <value>
                <![CDATA[Ivan]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[online_tusur.unit_online_tusur.Student@44e544f8]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [I] but found [Ivan]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [I] but found [Ivan]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at online_tusur.unit_online_tusur.TestFullStudent$GetFirstNameTest.test_getFirstName(TestFullStudent.java:49)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:77)
at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:15)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test_getFirstName -->
      </class> <!-- online_tusur.unit_online_tusur.TestFullStudent$GetFirstNameTest -->
      <class name="online_tusur.unit_online_tusur.TestFullStudent$AvgAgeTest">
        <test-method status="PASS" signature="avgAgeTest(int,[Lonline_tusur.unit_online_tusur.Student;)[pri:0, instance:online_tusur.unit_online_tusur.TestFullStudent$AvgAgeTest@1d805aa1, instance params:[]]" name="avgAgeTest" duration-ms="10" started-at="2022-02-28T04:54:25 YAKT" data-provider="avgAgeProvider" finished-at="2022-02-28T04:54:25 YAKT">
          <params>
            <param index="0">
              <value>
                <![CDATA[23]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[[Lonline_tusur.unit_online_tusur.Student;@5c887052]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- avgAgeTest -->
      </class> <!-- online_tusur.unit_online_tusur.TestFullStudent$AvgAgeTest -->
      <class name="online_tusur.unit_online_tusur.TestFullStudent$MinAgeTest">
        <test-method status="PASS" signature="minAgeProvider(int,[Lonline_tusur.unit_online_tusur.Student;)[pri:0, instance:online_tusur.unit_online_tusur.TestFullStudent$MinAgeTest@176f7f3b, instance params:[]]" name="minAgeProvider" duration-ms="1" started-at="2022-02-28T04:54:25 YAKT" data-provider="minAgeProvider" finished-at="2022-02-28T04:54:25 YAKT">
          <params>
            <param index="0">
              <value>
                <![CDATA[19]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[[Lonline_tusur.unit_online_tusur.Student;@1290ed28]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- minAgeProvider -->
      </class> <!-- online_tusur.unit_online_tusur.TestFullStudent$MinAgeTest -->
      <class name="online_tusur.unit_online_tusur.TestFullStudent$MaxAgeTest">
        <test-method status="PASS" signature="maxAgeTest(int,[Lonline_tusur.unit_online_tusur.Student;)[pri:0, instance:online_tusur.unit_online_tusur.TestFullStudent$MaxAgeTest@58740366, instance params:[]]" name="maxAgeTest" duration-ms="1" started-at="2022-02-28T04:54:25 YAKT" data-provider="maxAgeProvider" finished-at="2022-02-28T04:54:25 YAKT">
          <params>
            <param index="0">
              <value>
                <![CDATA[30]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[[Lonline_tusur.unit_online_tusur.Student;@7a3269f5]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- maxAgeTest -->
      </class> <!-- online_tusur.unit_online_tusur.TestFullStudent$MaxAgeTest -->
      <class name="online_tusur.unit_online_tusur.TestFullStudent$GetAgeTest">
        <test-method status="PASS" signature="test_getAge(int,online_tusur.unit_online_tusur.Student)[pri:0, instance:online_tusur.unit_online_tusur.TestFullStudent$GetAgeTest@2100d047, instance params:[]]" name="test_getAge" duration-ms="1" started-at="2022-02-28T04:54:25 YAKT" data-provider="ageProvider" finished-at="2022-02-28T04:54:25 YAKT">
          <params>
            <param index="0">
              <value>
                <![CDATA[18]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[online_tusur.unit_online_tusur.Student@46358dc5]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test_getAge -->
        <test-method status="FAIL" signature="test_getAge(int,online_tusur.unit_online_tusur.Student)[pri:0, instance:online_tusur.unit_online_tusur.TestFullStudent$GetAgeTest@2100d047, instance params:[]]" name="test_getAge" duration-ms="1" started-at="2022-02-28T04:54:25 YAKT" data-provider="ageProvider" finished-at="2022-02-28T04:54:25 YAKT">
          <params>
            <param index="0">
              <value>
                <![CDATA[50]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[online_tusur.unit_online_tusur.Student@6288f354]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [18] but found [50]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [18] but found [50]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:907)
at org.testng.Assert.assertEquals(Assert.java:917)
at online_tusur.unit_online_tusur.TestFullStudent$GetAgeTest.test_getAge(TestFullStudent.java:66)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:77)
at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:15)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test_getAge -->
      </class> <!-- online_tusur.unit_online_tusur.TestFullStudent$GetAgeTest -->
      <class name="online_tusur.unit_online_tusur.TestFullStudent$GetLastNameTest">
        <test-method status="FAIL" signature="test_getLastName(java.lang.String,java.lang.String)[pri:0, instance:online_tusur.unit_online_tusur.TestFullStudent$GetLastNameTest@5a3a1bf9, instance params:[]]" name="test_getLastName" duration-ms="1" started-at="2022-02-28T04:54:25 YAKT" data-provider="lastNameProvider" finished-at="2022-02-28T04:54:25 YAKT">
          <params>
            <param index="0">
              <value>
                <![CDATA[Ivanov]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[i]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [i] but found [Ivanov]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [i] but found [Ivanov]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at online_tusur.unit_online_tusur.TestFullStudent$GetLastNameTest.test_getLastName(TestFullStudent.java:32)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:77)
at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:15)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test_getLastName -->
        <test-method status="FAIL" signature="test_getLastName(java.lang.String,java.lang.String)[pri:0, instance:online_tusur.unit_online_tusur.TestFullStudent$GetLastNameTest@5a3a1bf9, instance params:[]]" name="test_getLastName" duration-ms="1" started-at="2022-02-28T04:54:25 YAKT" data-provider="lastNameProvider" finished-at="2022-02-28T04:54:25 YAKT">
          <params>
            <param index="0">
              <value>
                <![CDATA[Petrova-Sergeeva]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Petrova-Sergeeva]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Petrova-sergeeva] but found [Petrova-Sergeeva]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Petrova-sergeeva] but found [Petrova-Sergeeva]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at online_tusur.unit_online_tusur.TestFullStudent$GetLastNameTest.test_getLastName(TestFullStudent.java:32)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:77)
at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:15)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test_getLastName -->
      </class> <!-- online_tusur.unit_online_tusur.TestFullStudent$GetLastNameTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
